### ISSUE TITLE
[CDP] Cleanup Why might I have this debt additional info component

### ISSUE THREAD

**Issue by josephrlee** [department-of-veterans-affairs/va.gov-team#92812]
## Description

Enter a detailed description of the work - gherkin syntax below is acceptable.

Given: A veteran has an overpayment
When: They see their debt detail
Then: THey should have relevant additional info text

See [Slack thread for further context](https://dsva.slack.com/archives/G01KBHYEXV2/p1726012953247569)

## Tasks
- [x] Add all the variations to Figma for [CAIA to review](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88076)
- [x] Work with CAIA to get correct content
- [ ] Update code in the CDP and remove the FSR version

## Design and Front End sync on implementation of design
- [X] Required
- [ ] Complete
- [ ] Not Required

## Acceptance criteria
- [ ] Correct additional info component has been created and drafted
- [ ] PR with updated code has been created
- [ ] Separate PR for cleanup has been merged

## Testing
- [ ] N/A if non development work
- [ ] Testing passed and documented in this ticket based off the "Then" statement in the description

Testing procedure:
If additional testing steps or credentials to perform testing are needed list them here

---

## Ticket creation

<details><summary>Ticket requirements</summary>

### Required to be "Ready"
- [ ] Description written
- [ ] Tasks defined
- [ ] Acceptance criteria written
- [ ] Testing defined
- [ ] Add Labels
- [ ] Attach to an Epic
- [ ] Estimate Points
</details>

**Comment by mgleo** [comment]
This ticket will roll over.

**Comment by josephrlee** [comment]
We are still waiting on feedback from CAIA on which copy set to use. Once we have the copy updated, we can move forward with this ticket.

Will move this to blocked for now.

**Comment by dhill802** [comment]
Related ticket with comment asking for help: https://github.com/department-of-veterans-affairs/va.gov-team/issues/88076#issuecomment-2347382904

**Comment by mgleo** [comment]
@denisecoveyduc's response: 
The simple language that is currently used is correct. It should read "we make separate payments for..." 

<img width="501" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMlFhQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--eccfee34f96ae86cdb40c774dd4cc97eccb4e68e/Screenshot%202024-10-02%20at%2012.59.42%E2%80%AFPM.png" alt="Screenshot 2024-10-02 at 12.59.42 PM.png" />

**Comment by JunTaoLuo** [comment]
The PR has been merged and the work is complete.
